news/runtimewarning.rst

```diff
@@ -0,0 +1,23 @@
+**Added:**
+
+* Adds condition to fix `RuntimeWarning`
```

Comment:
this is not quite informational enough.  Also we only use Added for functionality.  Are you adding functionality?